File: eternal_return/src/main/java/org/EternalReturn/ERCharacter/Character/hart/Character_Hart.java
package org.EternalReturn.ERCharacter.Character.hart;

import org.EternalReturn.ERCharacter.ERCharacter;
import org.EternalReturn.ERPlayer.ERPlayer;

public class Character_Hart extends ERCharacter {

    public Character_Hart(ERPlayer player) {
        super(player);
        this.ActiveCooldownSeconds = 8;
        this.PassiveCooldownSeconds = 8;
        registerMonobehaviour(new Passive());
        registerMonobehaviour(new Active());
    }


    @Override
    public String getName() { return "hart"; }

}

File: eternal_return/src/main/kotlin/org/EternalReturn/ERCharacter/character/hart/Character_Hart.kt
package org.EternalReturn.ERCharacter.character.hart

import org.EternalReturn.ERCharacter.ERCharacter
import org.EternalReturn.ERPlayer.ERPlayer

class Character_Hart(player: ERPlayer) : ERCharacter(player) {
    init {
        this.ActiveCooldownSeconds = 8
        this.PassiveCooldownSeconds = 8
        registerMonobehaviour(Passive())
        registerMonobehaviour(Active())
    }


    override val name: String
        get() = "hart"
}